charts/primary-site/values.yaml

```diff
@@ -1,3 +1,8 @@
 globals:
   site_token:
   foxglove_api_url: https://api.foxglove.dev
+
+primary_site:
+  lakeStorageProvider: google_cloud
```

Comment:
Is it worth doing some nesting here:

```
primary_site:
  lake:
     storage_provider: google_cloud
     bucket_name: foxglove-lake
```

Also why is `primary_site` using underscores but the other variables camel case?